src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
```

Comment:
I wrote above what's wrong with this code, there is dead code in it https://en.wikipedia.org/wiki/Dead_code